Sources/NostrSDK/EventCreating.swift

```diff
@@ -34,17 +34,82 @@ public extension EventCreating {
     /// Creates a ``TextNoteEvent`` (kind 1) and signs it with the provided ``Keypair``.
     /// - Parameters:
     ///   - content: The content of the text note.
+    ///   - repliedEvent: The event being replied to.
+    ///   - mentionedEvents: The ``EventTag``s with `mention` markers for the mentioned events.
     ///   - subject: A subject for the text note.
     ///   - customEmojis: The custom emojis to emojify with if the matching shortcodes are found in the content field.
     ///   - keypair: The Keypair to sign with.
     /// - Returns: The signed ``TextNoteEvent``.
     ///
-    /// See [NIP-01 - Basic Event Kinds](https://github.com/nostr-protocol/nips/blob/master/01.md#basic-event-kinds)
```

Comment:
Titles keep getting changed in NIPs so I'm just leaving them off when we reference them in documentation.